datastore/metadata.go

```diff
@@ -0,0 +1,65 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-mongodb/v3/mongodb"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type DefaultDatasetMetadata struct {
+	// This ID refers to the Population Type
+	ID               string `json:"id" bson:"id"`
+	DefaultDatasetID string `json:"default_dataset_id" bson:"default_dataset_id"`
+	Edition          string `json:"edition" bson:"edition"`
```

Comment:
What is this Edition and Version? Is it for the DefaultDataset? If so they should be called DefaultDatasetVersion and DefaultDatasetEdition. Or be in their own struct 'DefaultDataset' (so they read as DefaultDataset.Version etc)